### ISSUE TITLE
arctos: still image has invalid access URI

### ISSUE THREAD

**Issue by MortenHofft** [gbif/model-material#108]
@dustymc
The access URI https://arctos.database.museum/media/42 is a webpage and not the image
And the descriptions (and other metadata) like `Taxidermy preparation in position at time of death.` isn't included

first row below:
```csv
digital_entity_id,digital_entity_type,access_uri,web_statement,format,license,rights,rights_uri,access_rights,rights_holder,source,source_uri,creator,created,modified,language,bibliographic_citation
https://arctos.database.museum/media/42,STILL_IMAGE,https://arctos.database.museum/media/42,,,,,,,,,,,,,,
```

**Comment by Jegelewicz** [comment]
It seems that what we want here is this - https://web.corral.tacc.utexas.edu/UAF/arctos/2009_08_04/bison.jp

**Comment by dustymc** [comment]
That's not something I can do without resolution in our internal issue. (And that should really come from the collections, but I suppose it's The Community's job to make that leap...)

**Comment by MortenHofft** [comment]
That returns `401 Unauthorized` so no that wouldn't work.

**Comment by tucotuco** [comment]
@MortenHofft @timrobertson100 The database field access_uri is a mapping of the Audiovisual Core (was Audubon Core) term [ac:accessURI](https://ac.tdwg.org/termlist/#ac_accessURI) and is so being properly used by Arctos. @MortenHofft The expectation of access via an accessURI to a resource with a media mimetype is not a reasonable assumption. The question is what to do about it when it is not, such as in this Arctos case.

**Comment by MortenHofft** [comment]
That sounds reasonable. So is the `format` then what tells me what to expect? So that I should expect a mime-type in that field that will tell me what the accessURI is?
Or how will a consumer know what to expect from that accessURI?  We need something, else we will not be able to show images.

**Comment by tucotuco** [comment]
@MortenHofft Yes, format is the field to tell you what to expect at the accessURI. See https://ac.tdwg.org/format/.
